### DISCUSSION TITLE
[menubutton] problemas con las versiones de las dependencias

### DISCUSSION THREAD

**Discussion by unicolas** [TiendaNube/nimbus-patterns#31] [Ideas]
Hola!

Incorporamos `menubutton` con la versión 1.1.0. Ahora que actualizamos `components` a la versión 4.0.0 y `styles` a la 8.0.0 y manteniendo las dependencias de `menubutton` (`components` 2.8.0 y `styles` 6.11.0) el estilo se rompe:

![image](https://user-images.githubusercontent.com/7072150/230076624-f6adeaae-fd5e-4d6d-974d-07bb126c9c66.png)

Con actualizarlo a la versión 1.1.2 tenemos problemas de espaciados:

![image](https://user-images.githubusercontent.com/7072150/230077010-4a8cfe8c-db18-4cb0-b3a0-a88107234f93.png)

esta versión todavía depende de `components` 3.0.0 y `styles` 7.0.0, donde entiendo que está el problema. La única forma de resolverlo es forzando resolutions para las dependencias con `"@nimbus-ds/components": "4.0.0"` y `"@nimbus-ds/styles": "8.0.0"`

![image](https://user-images.githubusercontent.com/7072150/230077761-fcac785d-f1b9-4917-8f72-a2c15499d43d.png)

No sé como lo vienen planificando, pero estaría bueno no tener mucho gap entre los releases de components y patterns.

Nota: [en la documentación](https://nimbus.nuvemshop.com.br/es-AR/documentation/patterns/menu-button) también se ve la falta de padding.

**Comment by juniorconquista** [comment]
@unicolas en primer lugar, muchas gracias por informar, en realidad identificamos este problema ayer y, de hecho, era un conflicto de dependencia de patrones.

Dicho esto, lanzamos una nueva versión de todos los componentes que corrige el problema mencionado, así que simplemente actualice las dependencias de acuerdo con las nuevas versiones a continuación:

- `@nimbus-ds/app-shell@1.2.0`
- `@nimbus-ds/callout-card@1.5.0`
- `@nimbus-ds/data-list@1.1.0`
- `@nimbus-ds/data-table@1.1.0`
- `@nimbus-ds/empty-message@1.1.0`
- `@nimbus-ds/formfield@1.4.0`
- `@nimbus-ds/interactive-list@1.1.0`
- `@nimbus-ds/layout@1.2.0`
- `@nimbus-ds/menu@1.2.0`
- `@nimbus-ds/menubutton@1.2.0`
- `@nimbus-ds/nav-tabs@1.1.0`
- `@nimbus-ds/page@1.2.0`
- `@nimbus-ds/productupdates@1.3.0`
- `@nimbus-ds/sidemodal@1.6.0`
- `@nimbus-ds/thumbnail-with-action@1.1.0`

También puede encontrar más detalles sobre la solución en nuestra página de lanzamientos: https://nimbus.tiendanube.com/documentation/releases

**Reply by unicolas** [reply]
Excelente, ahora sí. Gracias!
